src/security/KeyData.cc

```diff
@@ -35,6 +35,8 @@ Security::KeyData::loadX509CertFromFile()
 
     cert = Ssl::ReadX509Certificate(bio); // error detected/reported below
 
+    selfSigned = (X509_check_issued(cert.get(), cert.get()) == X509_V_OK);
```

Comment:
commits e662f09d and 6e56fe33

I get Amos CertIsIssuedBy,CertIsSelfSigned,CertSubjectName and fix them to use Certificate references.

Also I add the new Ssl::X509NameToSBuf function to print an OpenSSL X509_NAME (used to store certificate subject and issuer name) to an SBuf object. I tried to avoid memory allocations into this function because may used in places where speed is a requirement. Actually it is implemented twice please read comments inside this function implementation.